### ISSUE TITLE
Percent Fuel: HUD value different to Lobby value

### ISSUE THREAD

**Issue by craigo-** [pioneerspacesim/pioneer#4684]
### Observed behaviour
When the ship's fuel level is not completely full or completely empty, the percent fuel shown in the lobby is not the same as the percent fuel shown on the HUD. The value displayed on the HUD is always a little bit more.

### Expected behaviour
I think they should agree - unless I am missing something? 

### Steps to reproduce
1. Start on Mars (or anywhere)... The percent fuel displayed on the HUD is 100%
![image](https://user-images.githubusercontent.com/4467963/66121975-069e3000-e63b-11e9-8ca6-875ebb1325e3.png)

2. Go to the lobby, reduce fuel from 100% to 50%
![image](https://user-images.githubusercontent.com/4467963/66121585-03567480-e63a-11e9-9b34-5b999ba5423a.png)

3. Switch to world view and take note of the percent fuel displayed on the HUD (56%)
![image](https://user-images.githubusercontent.com/4467963/66121602-0cdfdc80-e63a-11e9-94f3-b71ea258c239.png)

Shouldn't they be the same? Or is the HUD value something else? The [wiki's description](https://pioneerwiki.com/wiki/Flight_UI) of the value is "It shows the % of the tanks"...

__My pioneer version (and OS):__
Pioneer 20190203 (latest release)
Pioneer master (6fd8f02f69f51ed772e6d0f6888491feba60dbf6), approx. 20190915
Windows 10 Enterprise 1809 x64

**Comment by nozmajner** [comment]
Hmm, maybe one of them is showing it as a percetangle of the total deltav and the other as percetangle of tank capacity? They should definitelly match.

**Comment by craigo-** [comment]
OK, I see... They're not actually measuring the same thing. Using the same test scenario:

- Fuel 100%, delta-v 9790 (100% of maximum delta-v)
- Fuel 50%, delta-v 5503 (56% of maximum delta-v)

I did have an understanding that a certain reduction in fuel resulted in a smaller, nonlinear reduction in delta-v, because overall mass is less... The difference between these two values sort of quantifies that.

I'll look into creating an account for the wiki and contribute a description update there.

Thanks :smile:

**Comment by nozmajner** [comment]
Yes, you are correct. The deltaV reduction is not linear to fuel mass, because the overal mass decreases as you burn fuel.

**Comment by impaktor** [comment]
I'm re-opening, as I think we should discuss this more, because it's certainly going to confuse more players, so we need to make sure it's addressed in the best way.

Main question is: do we want these numbers to be different?

- What makes most sense to show when landed (then we care about how much fuel we've loaded / will load)

- What makes sense in flight? (we want to know delta-V, fuel economy)

_If_ they should show different, then at least the mouse over tooltip should reflect that (i.e. need a pull request), + wiki edit, as mentioned.

**Comment by craigo-** [comment]
Knowing what I know now, I think the numbers should be different. I think what is displayed in each view is the correct metric (fuel load when docked, delta-v when in flight).

Happy to provide a PR; I believe I've found where the tooltip lives.

Something else I discovered however, is that the tooltip gets smaller when hovering over the darker unit (%, Mm/s):

![output_o29thQ](https://user-images.githubusercontent.com/4467963/66974037-529aab80-f0f6-11e9-908b-1125d29936fc.gif)

Not sure why this is... Could it be fixed? Also happy to create this as a separate issue if that makes more sense.
